unicore/comments/service/views/comments.py

```diff
@@ -86,5 +102,54 @@ def delete_comment(request, uuid, connection):
 
 
 '''
-TODO: Comment collection resource
+Comment collection resource
 '''
+
+
+@app.route('/comments/', methods=['GET'])
+@inlineCallbacks
+def list_comments(request):
+    columns = Comment.__table__.c
+    filter_expr = comment_filters.get_filter_expression(request.args, columns)
```

Comment:
The available filter fields are defined here: https://github.com/universalcore/unicore.comments/pull/5/files#diff-73d96a82277e220c4381e9de5ec498f5R17. Would you rather they are defined here or name each query string param explicitly?

My main problem with defining the filter params explicitly is that there are a lot of them. In this case 31 because it includes `<`, `>`, `like`, `in`, etc. for some fields. Flags has fewer obviously. I see us using most of these, and possibly more, in a management interface.